src/dbnode/encoding/proto/common.go

```diff
@@ -76,11 +76,17 @@ const (
 	// Single bit op codes that get encoded into the compressed stream and
 	// inform the iterator / decoder how it should interpret subsequent
 	// bits.
-	opCodeNoMoreDataOrTimeUnitChange = 0
-	opCodeMoreData                   = 1
+	opCodeNoMoreDataOrTimeUnitChangeAndOrSchemaChange = 0
```

Comment:
hahaha the single bit has a lot of meaning ;)

Reply:
We definitely have a preference for longer names (for better readability) so that comments can be focused just on the logic rather than identifying vars/etc.